src/task.js

```diff
@@ -0,0 +1,95 @@
+import * as dom from './getDomElements.js';
+
+class Task {
+  constructor(description) {
+    this.description = description;
+    this.completed = false;
+    this.id = 0;
+  }
+
+  static tasks = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+
+  updateList() {
+    // This method updates the web page
+    this.foo = 'eslint use this';
```

Comment:
- [ ] [Optional] this piece of code seems not to be doing anything and it shows up as part of the object saved in the local storage, you should take it out